client/client-lib/src/lib.rs

```diff
@@ -5,6 +5,7 @@ pub mod transaction;
 pub mod utils;
 pub mod wallet;
 
+use std::time::SystemTime;
```

Comment:
Clippy complaining about this. Should we just do this import if we're not in WASM?

Reply:
Seems like our CI job for WASM doesn't enforce rustfmt or clippy. Asked a question about this [here](https://github.com/fedimint/fedimint/issues/345#issuecomment-1221640543).